### ISSUE TITLE
targetplatform and installerName

### ISSUE THREAD

**Issue by exaV** [beryx/badass-runtime-plugin#73]
When using targetplatform and installerName the `jpackageImage` task crashes if the project's name is different from installerName.
It seems like installerName is used for finding the output of the jre task, when project.name should be used instead.

Example:
// project.name = "application"
```
runtime {
    targetPlatform("windows_x64") {
        setJdkHome(jdkDownload("https://cdn.azul.com/zulu/bin/zulu15.28.51-ca-jdk15.0.1-win_x64.zip"))
    }
    targetPlatform("anotherPlatform") {
        setJdkHome(jdkDownload("https://cdn.azul.com/zulu/bin/zulu15.28.51-ca-jdk15.0.1-win_x64.zip"))
    }
    jpackage {
        targetPlatformName ="windows_x64"
        imageName = "myApp"
        installerName = "myApp"
    }
}
```
Error:
`Specified runtime image directory runtime-image: ...\build\jre\myApp-windows_x64 does not exist`

Notes:
If the project was named "myApp" it would work.
It works as expected when I remove `targetPlatform`  and `targetPlatformName`
Expected:
I am able to set imageName and installerName to whatever regardless of the project's name.

**Comment by siordache** [comment]
Fixed in 1.12.1.

**Comment by exaV** [comment]
Thanks
